### ISSUE TITLE
Find + recover Ubuntu .enc encoding file used for Fontlab

### ISSUE THREAD

**Issue by sladen** [daltonmaag/ubuntu#31]
There is a project-wide `.enc` encoding file used by Fontlab, see:

* daltonmaag/ubuntu/issues/12#issuecomment-349243837

Upon import into Fontlab, this `.enc` is merged with the in-progress `.vfb`, and assists with UI alignment that can make the designer's Fontlab workflow easier.

Ideally this `.enc` can be found and checked into the source code so that designers using Fontlab can find everything they require in one place.

**Comment by anthrotype** [comment]
it's the "public.glyphOrder" in lib.plist.
You can extract it for loading the UFO in defcon, getting that list of strings from the ufo lib dictionary, then writing it out one line per item. Every line contains two space separated strings: the glyph name, and the glyph index (integer). Inline comments are prefixed with `%`. There must be a header that starts with the string `%%FONTLAB ENCODING: ` followed on the same line by a unique ID of 12 digits (we used to use the current time stamp YYYYMMDDHHMM), then a semicolon and the name of the encoding that will show up in the menu.

For example:

```
%%FONTLAB ENCODING: 201712051032; [DaMa] Ubuntu Encoding v1.000
.notdef 0
.null 1
CR 2
space 3
[...]
```

**Comment by sladen** [comment]
Or we could just find + include the file that already exists and already appears to have been part of the source for beginning a new weight.

**Comment by anthrotype** [comment]
yeah, but it might not be in sync with the one that's been exported as public.glyphOrder into the UFO sources. There have probably been several iterations of that encoding file over the years.
